components/TactileButton.vue

```diff
@@ -46,4 +50,8 @@ export default {
   color: #fff;
   font-weight: 500;
 }
+
+.large {
```

Comment:
Könnten wir später in die `typography.scss` datei ballern

Reply:
Würde da vielleicht eher analog zu dem Spacing-Kram `font-size`-Varianten anlegen.